### ISSUE TITLE
maybe typo  "S:L452D" could be "S:L452R" + addition of B.1.1.519?

### ISSUE THREAD

**Issue by tuspjo** [hodcroftlab/covariants#157]
Dear Dr. Hodcroft, 

I think there is a typo on this page https://covariants.org/variants/20A.S.478K
"These sequences have Spike mutations at positions S:L452D (see Variant20C/S:452R page for more details) and MutationS:P681, both of which impact antibody binding."

I think it should be S:L452R, not S:L452D

Unrelated: the variant B.1.1.519 was first detected a week ago in my lab, and now we see it multiple times a day. Would you consider adding it to the mutation list? the variant seems to be most prevalent in Mexico and the US, and have these mutations (within my amplicon ;)) S:T478K, S:D614G, S:P681H, S:T732A, and one silent S:S691S (TCC->TCT)

Best

Tue

**Comment by patbohn** [comment]
> "These sequences have Spike mutations at positions S:L452D (see Variant20C/S:452R page for more details) and MutationS:P681, both of which impact antibody binding."

Yes, I also just came across this, I believe it's in this file: 
https://github.com/hodcroftlab/covariants/blob/master/content/clusters/20A.S.154K.478K.md

**Comment by emmahodcroft** [comment]
Hi @tuspjo and @patbohn - thank you both! You're right, this should be 452R, not D. When typing these out it's not uncommon that I get the wrong amino acid as a typo, unfortunately! I appreciate your spotting it and letting me know :)

I've pushed this up to the preview site and should get it live very soon!

**Comment by emmahodcroft** [comment]
@patbohn Do you have a twitter account I can credit?

**Comment by emmahodcroft** [comment]
@tuspjo Is this the cluster you're referring to?
https://nextstrain.org/groups/neherlab/ncov/S.P681?c=gt-S_478&label=mlabel:20B/A23756G

It certainly looks interesting, and has some international spread, but it is still fairly small compared to some variants of interest. I'm not sure I'd add it to CoVariants just yet, but if you keep me updated I would be happy to see if it might be added if it continues to grow? 

I am curious though, I'll have a little look into it.

**Comment by tuspjo** [comment]
Yes, that's it. The B.1.617.2 first found in India also carry the T478K, but several other mutations allows distinction of the B.1.617.2 and B.1.1.519. 

Thank you!

**Comment by emmahodcroft** [comment]
No worries @patbohn ! :) 
 
 @tuspjo - here's the build ("S.732A" for the moment). Certainly a very distinct cluster, and some global spread (notable clusters in Germany, for example): https://nextstrain.org/groups/neherlab/ncov/S.732A
 
 Zoomed to the cluster & coloured by country:
 https://nextstrain.org/groups/neherlab/ncov/S.732A?c=country&label=mlabel:20B/C203T&r=country

Do we know anything more about this variant? Are there any papers on it?

**Comment by tuspjo** [comment]
Yes. My lab does Sanger detection on a single amplicon bc it's dirt cheap&easy and can distinguish all known important variants, but it means that our data takes a bit to trickle through the various WGS pipelines before being deposited at GISAID, so Denmark is still listed as only having few B.1.1.519s. 
Currently, there's a confirmed outbreak of the B.1.1.519 here, so likely >100 WGS sequences will be coming the next weeks from Denmark, as all positive samples are WGSed.

**Comment by tuspjo** [comment]
I have not found much info on this variant, it's the dominating strain in Mexico, but only a few newspaper articles are mentioning it so far I think, and this preprint on T478K (which I think does not relate to the fact that the B.1.617.2 (Indian-2) also carries the T478K, maybe for timing reasons as B.1.617.2 is fairly recent): https://www.biorxiv.org/content/10.1101/2021.03.28.437369v3.full.pdf+html
I think B.1.1.222 is a ancester of B.1.1.519, at least they share many mutations.
